backend/EasyMeets.Core/EasyMeets.Core.WebAPI/Controllers/QueueController.cs

```diff
@@ -0,0 +1,27 @@
+﻿using EasyMeets.Core.BLL.Interfaces;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace EasyMeets.Core.WebAPI.Controllers
+{
+    //[Authorize]
+    [ApiController]
+    [Route("[controller]")]
+    public class QueueController : ControllerBase
+    {
+        private readonly IInformQueueService _queueService;
+
+        public QueueController(IInformQueueService queueService)
+        {
+            _queueService = queueService;
+        }
+
+        [HttpPost("InformEveryone")]
+        public ActionResult InformEveryone(string message)
```

Comment:
Is it just for testing?

Reply:
deleted this controller , just tested API and have in Trello card "API will write to the queue" thought we need to have some controller to send message from API to queue